controller/array_action/svc_cli_result_reader.py

```diff
@@ -0,0 +1,117 @@
+from controller.common.csi_logger import get_stdout_logger
+import controller.array_action.errors as controller_errors
+
+logger = get_stdout_logger()
+
+ID_PARAM_NAME = "id"
+
+
+class SVCListResultsReader:
+    """
+    Iterable object used to read raw command response from m SVS array.
+    Each object in response is translated to SVCListResultsElement.
+    Input is received as string which represents '\n'-separated list of returned lines from output.
+    Line with param 'id' (e.g. 'id 1') is recognized as first line of object ans used as separator between objects
+    (e.g. in input "id 1\nname n3\nid 2\nname n2" first object starts with line 'id 1' and ends with 'id 2')
```

Comment:
It is hard to understand the comment with the "\nn"

Reply:
because it is \nname or all the comment?
If i'll use other example it ill be ok? E.g. \nwwns

Reply:
but this is the exact example. Lines with \n char separator
Do u gave any other idea how to write the example? I can do this way  but less like it
"id 1
name name1
id 2
name name2"

Reply:
@roysahar-ibm how do u wan t me to change it?

Reply:
Replace the "\n" with "<New Line>"

Reply:
but the input contains nerwline

Reply:
Will use <new line>